src/copilot/tests/test_service_source.py

```diff
@@ -0,0 +1,82 @@
+import os
+import pytest
+from testcontainers.postgres import PostgresContainer
```

Comment:
src/copilot/tests/test_service_source.py:3: in <module>
    from testcontainers.postgres import PostgresContainer
E   ModuleNotFoundError: No module named 'testcontainers'

Need to install in requirements.txt?

Reply:
It was already in requirements.txt, but I just moved pytest and testcontainers out of the regular requirements.txt and to a new tests/requirements.txt file.